vrc-get-gui/src/commands/environment/templates.rs

```diff
@@ -200,6 +200,9 @@ async fn save_template_file(
 
     let mut file = 'create_file: {
         let template_dir = Path::new("vrc-get/templates");
+        if !io.is_dir(template_dir).await {
+            io.create_dir_all(template_dir).await?;
+        }
```

Comment:
```suggestion
        io.create_dir_all(template_dir).await?;
```

create_dir_all doesn't throw any error if already exists so checking for directory can be removed.